spec/genre_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../classes/genre'
+
+describe Genre do
+  before :each do
+    @genre = Genre.new('country')
+  end
+
+  it 'test if object is instance of correct class' do
+    expect(@genre).to be_an_instance_of Genre
+  end
+
+  it 'genre name should be correct' do
+    @name == 'country'
+  end
+
+  it 'The add_items method should add item to @@items' do
+    item1 = Item.new(Date.today.to_s, true)
+    @genre.add_item(item1)
+    items = Genre.class_variable_get(:@@items)
+    expect(items).to include(item1)
+  end
+end
```

Comment:
- [ ]  Good job adding this test. However, this test is failing, let's try to fix it.

<img width="469" alt="chrome_55BBV8cX36" src="https://user-images.githubusercontent.com/81468675/154713833-8fefb69a-dfc3-4827-a412-7a5cb6b0e671.png">